### PR TITLE
GPU augs NormalizeTorch and ISONoiseTorch

### PR DESCRIPTION
**Pull request by Dipet**

### CODE REVIEW

File: albumentations/pytorch/transforms.py [hunk 1]
```diff
@@ -82,21 +83,39 @@ def get_transform_init_args_names(self):
...
 
     def get_transform_init_args_names(self):
-        return []
+        return ("device",)
 
     def get_params_dependent_on_targets(self, params):
         return {}
+
+
+class BasicTransformTorch(BasicTransform):
```
Comment (vfdev-5) [inline albumentations/pytorch/transforms.py]: Is it intended to have no docstring for a public class ?

File: albumentations/pytorch/transforms.py [hunk 2]
```diff
@@ -82,21 +83,39 @@ def get_transform_init_args_names(self):
 class ToTensorV2(BasicTransform):
     """Convert image and mask to `torch.Tensor`."""
 
-    def __init__(self, always_apply=True, p=1.0):
+    def __init__(self, always_apply=True, device="cpu", p=1.0):
```
[the remark this answers is not in the document]
Comment (Dipet) [reply albumentations/pytorch/transforms.py]: It is need for GPU augs

[the remark this answers is not in the document]
Comment (vfdev-5) [reply albumentations/pytorch/transforms.py]: So, integration with Kornia is on its way ?

[the remark this answers is not in the document]
Comment (Dipet) [reply albumentations/pytorch/transforms.py]: Yes, I use some features from kornia, but some transforms has been rewriten to speedup code. So, differentiability is not guaranteed

**Comment by Dipet** [comment]
Current implementation does not give any speedup and has problems with multiprocessing.

**Pull request closed without merging**